tox.ini

```diff
@@ -8,6 +8,15 @@ usedevelop = True
 extras =
 	testing
 
+# Check whether there's dependencies present to even make the script run.
+[testenv:check_depends]
+deps =
+extras =
```

Comment:
No need for these if we're not giving them values

```suggestion
```

Reply:
Ahh okay, I thought setting them empty was overriding the defaults set in the top-level `testenv` section of `tox.ini`.  I saw `deps =` there already, so thought it had that specific meaning.

Reply:
ohhh wait I think you're right, sorry!

Though I'd say technically the proper way to fix that is to make the top-level section header into `[testenv:py,py3{6,7,8,9,10,11,12}]` so that it explicitly lists environments, rather than defining defaults. But if you don't feel like getting into that as part of this PR, no worries.